src/components/vm/vmDetailsPage.jsx

```diff
@@ -107,6 +108,23 @@ export const VmDetailsPage = ({
         );
     }
 
+    const [selectedConsole, setSelectedConsole] = React.useState(() => {
+        if (vm.displays) {
```

Comment:
This is different from the old `getDefaultConsole`. We used to default to "vnc", then "spice", then "serial", in that order. Now we prefer "serial". Is that intentional?